Gemfile

```diff
@@ -23,7 +23,6 @@ group :development do
   gem "kramdown"
   gem 'simplecov'
   gem "codeclimate-test-reporter"
-  gem 'jruby-lint'
```

Comment:
:+1: removing jruby-lint, it's a gem to detect incompatible gems and bad usage patterns etc... so it's nice but hasn't been updated since 2013 :( Also so far it never reported anything for us